src/components/Button/Button.js

```diff
@@ -0,0 +1,21 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+import StyledButton from './StyledButton';
+
+function Button({ label, children, className }) {
```

Comment:
First I think we want these buttons to be used as regular HTML button element, so we don't need label (text would be passed as a child):
```
function Button({ children, icon, className }) {
  return <StyledButton>{children}{icon}</StyledButton>
}
```
And afterwards icon would take care about positioning itself.